### PR TITLE
Feature/oeo steering committe subpage

### LINKED ISSUE #604: Add OEO Steering Committee Subpage
The OEO Steering Committee needs its own web page, which should be a sub page of the OEP. Please create such a sub page @jh-RLI . I think it makes sense to link it somewhere under ontology.

https://openenergy-platform.org/ontology/

The URL would then be 

https://openenergy-platform.org/ontology/oeo-steering-committee

Content for the page is here:

https://github.com/OpenEnergyPlatform/ontology/wiki/OEO-Steering-Committee

An issue to create an English translation is open here: https://github.com/OpenEnergyPlatform/ontology/issues/313

Creating the page and making it look simple, but decent enough are priorities. The final text and location can easily be changed later on. Contact me if you have any questions. 

Feel free to give feedback make changes to this issue @Ludee

### PR DESCRIPTION
**Pull request by jh-RLI**
- New subpage for oeo-steering committee
- Add content

closes #604

### CODE REVIEW

**Comment by jh-RLI** [comment]
@christian-rli the subpage exists and is accessible via the ontology about page. Is someone working on the translation of the contend?

**Comment by christian-rli** [comment]
@jh-RLI no one has been working on these yet. I'll do it as soon as I get to it.

**Comment by christian-rli** [comment]
@jh-RLI I translated the page, but content is missing or is partly redundant with what is available already on the ontology explanation page. 

@Ludee did you get confirmation from the partners, so we can list them?

File: ontology/templates/ontology/about.html [hunk 1]
```diff
@@ -53,6 +53,6 @@ <h3 id="where-and-how-can-i-contribute">Where and how can I contribute?</h3>
 <p>The OEO development was initiated in the project “SzenarienDB” and is
     augmented in the project “LOD_GEOSS”. The development is community based and
     <a href="https://github.com/OpenEnergyPlatform/ontology">takes place on GitHub</a>. </p>
-<p>A OEO steering committee was created in autumn 2019.</p>
+<p>A  <a role="button" class="btn btn-secondary btn-sm" href={% url 'oeo-s-c'%}>OEO steering committee</a> was created in autumn 2019.</p>
```
Comment (MGlauer) [inline ontology/templates/ontology/about.html:56]: Why is this a button?

Comment (jh-RLI) [reply ontology/templates/ontology/about.html:56]: atm this is the only link to the oeo-s-c page, I wanted to highlight it with the button

Comment (MGlauer) [reply ontology/templates/ontology/about.html:56]: I think we should stick with a normal link here

Comment (jh-RLI) [reply ontology/templates/ontology/about.html:56]: okay

**Pull request merged**